gensim/models/keyedvectors.py

```diff
@@ -1063,6 +1067,139 @@ def get_keras_embedding(self, train_embeddings=False):
         )
         return layer
 
+    def simple_average(self, sent):
+        """
+        Mixture of word-vectors to make sentence-vector.
+
+        Attains this by simple averaging of all word-vectors.
+
+        Examples
+        --------
+        >>> from gensim.models import KeyedVectors
+        >>> trained_model = KeyedVectors.load_word2vec_format("glove-wiki-gigaword-200.gz")
+        >>> sent = [['sample', 'sentence', 'cat', 'dog'], ['time', 'computers', 'expensive']]
+        >>> trained_model.simple_average(sent)
+
+        Parameters
+        ----------
+         sent : list iterator
+            iterator of list which contains tokens of sentences.
+
+        Returns
+        -------
+        sents_emd : class : `gensim.models.keyedvectors.Word2VecKeyedVectors`
+            Sentence embeddings of each sentence in the same order as given.
+
+        """
+        sents_emd = []
+        for s in sent:
```

Comment:
if I pass one sentence - this didn't work

Reply:
We talked about this since SIF and TFIDF need full corpus and to make it equivalent to all three mixture methods need full corpus, not just one sentence.